drivers/extend/include/gpio_ext_conf.h

```diff
@@ -0,0 +1,63 @@
+/*
+ * Copyright (C) 2018 Acutam Automation, LLC
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @ingroup     drivers_extend_gpio
+ *
+ * @{
+ *
+ * @file
+ * @brief       GPIO extension device default / example list
+ *
+ * If the `extend_gpio` module is enabled, a list of GPIO extension devices
+ * of type @ref gpio_ext_dev_t has to be provided as a configuration.
+ *
+ * This file defines a default list containing only the device with the
+ * default not-supported GPIO extension functions `gpio_ext_*_notsup`.
+ *
+ * The board has to override this file with the real GPIO extension device
+ * list in file `boards/.../include/gpio_ext_conf.h`.
+ *
+ * @author      Matthew Blue <matthew.blue.neuro@gmail.com>
+ *
+ * @}
+ */
+
+#ifndef GPIO_EXT_CONF_H
+#define GPIO_EXT_CONF_H
+
+#include <stddef.h>
+
+#include "extend/gpio.h"
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+/**
+ * @brief   Reference the driver struct
+ */
+extern const gpio_ext_driver_t gpio_ext_notsup_driver;
+
+/**
+ * @brief   GPIO expansion default list if not defined
+ */
+static const gpio_ext_dev_t gpio_ext[] =
+{
+    {
+        .driver = &gpio_ext_notsup_driver,
```

Comment:
I don't get this. Unless testing the GPIO extension API, would an empty list of drivers be better in case no drivers are provided?

Reply:
`gpio_ext` has to be overriden by real configuration if the extension API is used.